src/lib/contains.js

```diff
@@ -4,12 +4,14 @@ import merge from './util/merge';
 
 const defaulContainsOptions = {
   ignoreCase: false,
+  minOccurrences: 1,
 };
 
 export default function contains(str, elem, options) {
   assertString(str);
   options = merge(options, defaulContainsOptions);
-  return options.ignoreCase ?
-    str.toLowerCase().indexOf(toString(elem).toLowerCase()) >= 0 :
-    str.indexOf(toString(elem)) >= 0;
+
+  const regex = new RegExp(toString(elem), `g${options.ignoreCase ? 'i' : ''}`);
+
+  return (str.match(regex) || []).length >= options.minOccurrences;
```

Comment:
Do we really need a regex here? I really don't like regexes based on user input since it may introduce ReDos
What do you think about using `split` instead?

```suggestion
  if(options.ignoreCase) {
     return str.toLowerCase().split(toString(elem).toLowerCase()).length > options.minOccurences;
  }
  return str.split(toString(elem)).length > options.minOccurences
```

Reply:
Good point, replaced regex with split now.